Dshop/apps/payments/views.py

```diff
@@ -13,3 +14,68 @@ def stripe_config(request):
     if request.method == 'GET':
         stripe_config = {'publicKey': settings.STRIPE_PUBLISHABLE_KEY}
         return JsonResponse(stripe_config, safe=False)
+
+
+@csrf_exempt
+def create_checkout_session(request):
+    if request.method == 'GET':
+        domain_url = 'http://localhost:8000/payments/'
```

Comment:
Do not hardcode domain or protocol.  This will for example not run on 127.0.0.1 
1. Use `reverse` fumction to create urls
2. Protocol and domain has to be set dynamically. One way to do it is to use sites framework https://docs.djangoproject.com/en/4.2/ref/contrib/sites/